src/components/Profile.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { Link, useStaticQuery, graphql } from 'gatsby';
+import Espai from './../images/espai.png';
+import styled from 'styled-components';
+import { Parallax } from 'react-scroll-parallax';
+import { Container, Blobs, BlobMask, SectionHeader } from './../styles';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPenFancy } from '@fortawesome/free-solid-svg-icons';
+
+const ProfileContainer = styled(Container)`
+  align-items: center;
+  justify-content: center;
+  padding: -4rem 2rem;
+
+  @media ${({ theme }) => theme.device.tablet} {
+    padding: ${({ theme }) => theme.padding.tablet};
+    flex-direction: row;
+  }
+  @media ${({ theme }) => theme.device.laptop} {
+    padding: ${({ theme }) => theme.padding.laptop};
+  }
+`;
+
+const ImageMask = styled(BlobMask)`
+  mask-image: url(${Blobs.profile});
+`;
+
+const TextContainer = styled.div`
+  margin-top: -11rem;
+  margin-left: 1.5rem;
+  display: flex;
+  flex-direction: column;
+  color: ${({ theme }) => theme.color.accent};
+  z-index: 1;
+
+  @media ${({ theme }) => theme.device.tablet} {
+    margin-top: 5rem;
+    width: 35rem;
+  }
+`;
+
+const StyledParallax = styled(Parallax)`
+  @media ${({ theme }) => theme.device.tablet} {
+    margin-left: -5rem;
+  }
+`;
+
+const Titles = styled.p`
+  padding: 0 1rem;
+`;
+
+const PostSpotlight = styled.div`
+  padding: ${({ theme }) => theme.padding.mobile};
+`;
+
+interface ProfileProps {
+  id: string;
+}
+
+const Profile = ({ id }: ProfileProps) => {
+  const data = useStaticQuery(graphql`
```

Comment:
I recommend passing an interface to your useStaticQuery function so you can shape the response. As it stands, your IDE is probably inferring data to be of type `any`. passing an interface will allow you to destructure your response and simplify some of the code below.

first create your interface:
```typescript
interface Frontmatter {
    description: string;
    slug: string;
}

interface MarkdownRemark {
    frontmatter: FrontMatter;
}

interface MyResponse {
    markdownRemark: MarkdownRemark;
}
```

```suggestion
  const const {
    markdownRemark: {
      frontmatter: { description, slug },
    },
  } = useStaticQuery<MyResponse>(graphql`
```

note: up to you what you want to destructure.